fatal-errors.json

```diff
@@ -64,6 +64,7 @@
     "GeodeticCoordinateSystemNotSupported": "https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/48731/geoconnector#GeodeticCoordinateSystemNotSupported",
     "IModelAccessForbidden": "https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/54636/knowledge-base#IModelAccessWasForbidden",
     "IModelNotFound": "https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/54636/knowledge-base#iModelName",
+    "IncompleteDxfFile" : "https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/54636/knowledge-base#IncompleteDxfFile",

```

Comment:
What about a name like "DxfMissingHeader"? I know it's a slippery slope, but there may be other ways a DXF is incomplete that we'll need to uniquely identify in theory.

Reply:
Incomplete header will cover more cases.  If this same user adds a header but still does not have codepage set, we will run into the same problem.

Reply:
Sure... "IncompleteDxfHeader" works for me... still more future-proof than IncompleteDxfFile.

Reply:
Renamed the key